addons/event_sale/wizard/event_configurator.py

```diff
@@ -11,3 +12,34 @@ class EventConfigurator(models.TransientModel):
     product_id = fields.Many2one('product.product', string="Product", readonly=True)
     event_id = fields.Many2one('event.event', string="Event")
     event_ticket_id = fields.Many2one('event.event.ticket', string="Event Ticket")
+    has_available_events = fields.Boolean(
```

Comment:
Can you add a string "Has Available Events" as first argument to set the name of the field?
It does it automatically based on the variable name but I'm not a fan.

Reply:
argument added!